content/php-client/resources/PAM/asset-reference-files.md

```diff
@@ -58,6 +58,13 @@ $product = $client->getAssetReferenceFileApi()->downloadFromLocalizableAsset('ch
 file_put_contents('/tmp/chicagoskyline.jpg', $product->getContents());
 ```
 
+From v4 of the PHP client, the response is returned instead of the content. It allows to get the filename and the Mime type from the response.
```

Comment:
```suggestion
From the v4 of the PHP client, the response is returned instead of the content. It allows getting the filename and the MIME type from the response.
```